### ISSUE TITLE
Should Android setup dialog have a title bar close button?

### ISSUE THREAD

**Issue by dbjorge** [microsoft/accessibility-insights-web#2968]
## Describe the bug

As-designed, the new Accessibility Insights for Android device setup flow is being styled like a modal dialog without the "normal" title bar close/minimize/maximize buttons. In the earlier version, one of the reasons we did this was because there was always an explicit "Close" button as part of the dialog itself, but in the newer/more complex flow, there are many states where there is only a "Cancel" button that moves to a previous step, and no button anywhere on the dialog that closes the dialog.

I'm not sure it's an accessibility violation, exactly, but https://www.w3.org/TR/wai-aria-practices/#dialog_modal notes that "It is strongly recommended that the tab sequence of all dialogs include a visible element with role `button` that closes the dialog, such as a close icon or cancel button".

We decided not to block the android-setup feature over this question; this bug tracks wanting to gather more information/feedback from our designers before proceeding with any changes. Options might include:

* "use the same OS-native title bar as the automated checks view"
* "use a more minimal title bar with a stylized 'x' icon"
* "adjust the action buttons so there's always a close option"
* "it's okay as-is, leave it alone"

@ferBonnin has offered to discuss this with our design folks at their next office hours

**Comment by ferBonnin** [comment]
Per conversation with Peter, we will explore this and also look for user feedback

**Comment by ferBonnin** [comment]
Per conversation with @peterdur closing this, no user feedback on this
